gateway-service-knoxsso/src/main/java/org/apache/knox/gateway/service/knoxsso/WebSSOResource.java

```diff
@@ -241,6 +249,18 @@ private Response getAuthenticationToken(int statusCode) {
     Principal p = request.getUserPrincipal();
 
     try {
+      Map<String, String> claims = new HashMap<>();
+      int cookieAge = maxAge;
+      Optional<String> awsCookie =  CookieUtils.getCookieValue(request, AwsConstants.AWS_COOKIE_NAME);
+      AwsSamlCredentials awsSamlCredentials = null;
+      if (awsCookie.isPresent()) {
+        String cookieJson = new String(Base64.decodeBase64(awsCookie.get().getBytes("UTF-8")), "UTF-8");
```

Comment:
Should probably use `StandardCharsets...` instead of "UTF-8" if possible.

Reply:
Done